### ISSUE TITLE
Deploy services using SBT Docker deployer module

### ISSUE THREAD

**Issue by fernandohackbart** [fernandohackbart/bikeride-lagom#129]
Each Service becomes a docker image?

https://www.lagomframework.com/documentation/1.3.x/scala/ProductionOverview.html

**Comment by fernandohackbart** [comment]
It generates a container image that does not put the executable in background...

Need to understand better the defaults

**Comment by fernandohackbart** [comment]
```
lazy val dockerSettings = Seq(packageName in Docker := "bikeride-backend",
  packageName in Docker := "bikeride-backend",
  //aggregate in Docker := true
  packageSummary in Docker := "BikeRide Backend in Lagom",
  packageDescription in Docker := "BikeRide Backend in Lagom",
  version in Docker := "0.0.1",
  maintainer in Docker := "Fernando Hackbart<fhackbart@gmail.com>",
  dockerBaseImage := "bikeride/bikeride-backend:base",
  daemonUser in Docker := "bikeride",
  dockerExposedPorts := Seq(9000,8000,54610,61817,54431,49454),
  //dockerExposedUdpPorts in Docker :=
  //dockerExposedVolumes in Docker :=
  //dockerLabels in Docker :=
  dockerEntrypoint in Docker := Seq("/u01/bikeride/bin/bikeride-lagom"),
  //defaultLinuxInstallLocation in Docker := "/u01/bikeride",
  dockerRepository in Docker := Some("bikeride")
  //dockerUsername in Docker :=
  //dockerUpdateLatest in Docker :=
  //dockerAlias in Docker :=
  //dockerBuildOptions in Docker :=
  //dockerExecCommand in Docker := Seq("sudo","/usr/bin/docker")
  //dockerBuildCommand in Docker :=
  //dockerRmiCommand in Docker :=
  )
```

**Comment by fernandohackbart** [comment]
```
lazy val `bikeride-lagom` = (project in file("."))
  .aggregate(`authentication-lagom-api`,
    `authentication-lagom-impl`,
    `biker-lagom-api`,
    `biker-lagom-impl`,
    `track-lagom-api`,
    `track-lagom-impl`,
    `ride-lagom-api`,
    `ride-lagom-impl`,
    `utils`)

lazy val utils = (project in file("utils"))
  .settings(
    libraryDependencies ++= Seq(
      lagomScaladslApi,
      lagomScaladslServer % Optional,
      scalaTest,
      jwt
    )
  )

lazy val `authentication-lagom-api` = (project in file("authentication-lagom-api"))
  .settings(
    libraryDependencies ++= Seq(
      lagomScaladslApi
    )
  )
  .dependsOn(`utils`,`biker-lagom-api`)

lazy val `authentication-lagom-impl` = (project in file("authentication-lagom-impl"))
  .enablePlugins(LagomScala)
  .settings(
    libraryDependencies ++= Seq(
      lagomScaladslPersistenceCassandra,
      lagomScaladslKafkaBroker,
      lagomScaladslTestKit,
      macwire,
      scalaTest
    ),
    dockerSettings,
    lagomServicePort := 9010
  )
  .settings(lagomForkedTestSettings: _*)
  .dependsOn(`authentication-lagom-api`,`biker-lagom-api`,`utils`)

lazy val `biker-lagom-api` = (project in file("biker-lagom-api"))
  .settings(
    libraryDependencies ++= Seq(
      lagomScaladslApi
    )
  )
  .dependsOn(`utils`)

lazy val `biker-lagom-impl` = (project in file("biker-lagom-impl"))
  .enablePlugins(LagomScala)
  .settings(
    libraryDependencies ++= Seq(
      lagomScaladslPersistenceCassandra,
      lagomScaladslKafkaBroker,
      lagomScaladslTestKit,
      serviceLocatorDNS,
      macwire,
      scalaTest
    ),
    dockerSettings,
    lagomServicePort := 9011
  )
  .settings(lagomForkedTestSettings: _*)
  .dependsOn(`biker-lagom-api`,`utils`)

lazy val `track-lagom-api` = (project in file("track-lagom-api"))
  .settings(
    libraryDependencies ++= Seq(
      lagomScaladslApi
    )
  )
  .dependsOn(`utils`)

lazy val `track-lagom-impl` = (project in file("track-lagom-impl"))
  .enablePlugins(LagomScala)
  .settings(
    libraryDependencies ++= Seq(
      lagomScaladslPersistenceCassandra,
      lagomScaladslKafkaBroker,
      lagomScaladslTestKit,
      macwire,
      scalaTest
    ),
    dockerSettings,
    lagomServicePort := 9012
  )
  .settings(lagomForkedTestSettings: _*)
  .dependsOn(`track-lagom-api`,`biker-lagom-api`,`utils`)


lazy val `ride-lagom-api` = (project in file("ride-lagom-api"))
  .settings(
    libraryDependencies ++= Seq(
      lagomScaladslApi
    )
  )
  .dependsOn(`utils`,`biker-lagom-api`,`track-lagom-api`)


lazy val `ride-lagom-impl` = (project in file("ride-lagom-impl"))
  .enablePlugins(LagomScala)
  .settings(
    libraryDependencies ++= Seq(
      lagomScaladslPersistenceCassandra,
      lagomScaladslKafkaBroker,
      lagomScaladslTestKit,
      macwire,
      scalaTest
    ),
    dockerSettings,
    lagomServicePort := 9013
  )
  .settings(lagomForkedTestSettings: _*)
  .dependsOn(`ride-lagom-api`,`track-lagom-api`,`biker-lagom-api`)
```

**Comment by fernandohackbart** [comment]
This is not working as expected and was solved by #134
